src/lib/apis/ollama/index.ts

```diff
@@ -249,8 +249,7 @@ export const deleteModel = async (token: string, tagName: string) => {
 };
 
 export const pullModel = async (token: string, tagName: string) => {
```

Comment:
_Note:_ Refactored this because `res` would never be `null` because we explicitly assign `err` to `error` and then throw it from line 269. With the refactor, the function's return type has changed from `Promise<Response | null>` to `Promise<Response>` making it easier to read / use and no need `if` to check if `res` is valid.

Reply:
Will be reverting to the old code, as it should also catch and raise error when the backend proxy can't access Ollama.

Reply:
<img width="836" alt="image" src="https://github.com/ollama-webui/ollama-webui/assets/25473318/8f100e17-5997-403b-8e0e-f929d7b2e0b4">

Updated to display this error message